rel/jenkins/percona-create-repos.groovy

```diff
@@ -37,19 +37,37 @@ pipeline {
     }
     stages {
         stage('Create YUM repo') {
-            steps {
-                createRepo(REPO_NAME, 'yum', COMPONENTS, CENTOS_VERSIONS, DEB_CODE_NAMES, LIMIT)
-            }
+             steps {
+                script {
+                    if (REPO_NAME.endsWith("pro")) {
```

Comment:
We have got not only "-pro" repos in private section.
Could it be added a selector for choosing a common or private repo?

Reply:
Thank you for the review comment. I have made the necessary changes.